elm.json

```diff
@@ -25,7 +25,7 @@
       "jzxhuang/http-extras": "2.1.0",
       "krisajenkins/remotedata": "6.0.1",
       "pablen/toasty": "1.2.0",
-      "ryannhg/date-format": "2.3.0",
+      "ryan-haskell/date-format": "1.0.0",
```

Comment:
nice, it got renamed, but i think this should still be `2.3.0` https://github.com/ryan-haskell/date-format/releases

Reply:
The repo owner suggested using `v1.0.0` in this thread https://discourse.elm-lang.org/t/ryannhg-packages-renamed-to-ryan-haskell/9705

Reply:
excellent! ignore me!